PCbuild/build.bat

```diff
@@ -116,7 +116,7 @@ if "%kill%"=="true" call :Kill
 
 if "%do_pgo%"=="true" (
     set conf=PGInstrument
-    call :Build
+    call :Build %1 %2 %3 %4 %5 %6 %7 %8 %9

```

Comment:
I'm a little concerned about the `^M` disappearing from the line. Batch files are very fussy about line endings.

With confirmation that this doesn't break the line ending, I'm good with the fix.